udata/api/fields.py

```diff
@@ -4,7 +4,28 @@
 import pytz
 from dateutil.parser import parse
 from flask import request, url_for
-from flask_restx.fields import *  # noqa
```

Comment:
Is this due to ruff complaint? Shouldn't it be bypass with a #noqa mechanic?

Reply:
It's bypassed on this line, but as explained above, it'll then complain about all the `String` and other `flask_restx` fields in the file, not being able to make sure it's coming from `flask_restx.fields`: https://docs.astral.sh/ruff/rules/undefined-local-with-import-star-usage/